wnbd-build/build/build

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
+set -o errexit
+set -o pipefail
+
+BUILD_CONFIGURATION=${BUILD_CONFIGURATION:-"Release"}
+
+
+#
+# Install requirements (if needed)
+#
+if ! which zip >/dev/null; then
+    sudo apt-get update
+    sudo apt-get install -y zip
+fi
+
+#
+# Upload wnbd repo to the Windows VM
+#
+scp_upload $WORKSPACE/wnbd /workspace/wnbd
+
+#
+# Build the Visual Studio project
+#
+BUILD_CMD="MSBuild.exe %SystemDrive%\\workspace\\wnbd\\vstudio\\wnbd.sln /p:Configuration=${BUILD_CONFIGURATION}"
+SSH_TIMEOUT=30m ssh_exec "\"%ProgramFiles(x86)%\\Microsoft Visual Studio\\2019\\BuildTools\\VC\\Auxiliary\\Build\\vcvarsall.bat\" x86_amd64 & ${BUILD_CMD}"
+
+#
+# Download the build artifacts
+#
+mkdir -p $WORKSPACE/build/wnbd/driver
+scp_download /workspace/wnbd/vstudio/x64/${BUILD_CONFIGURATION}/driver/* $WORKSPACE/build/wnbd/driver/
+
+mkdir -p $WORKSPACE/build/wnbd/binaries
+scp_download /workspace/wnbd/vstudio/x64/${BUILD_CONFIGURATION}/libwnbd.dll $WORKSPACE/build/wnbd/binaries/
+scp_download /workspace/wnbd/vstudio/x64/${BUILD_CONFIGURATION}/wnbd-client.exe $WORKSPACE/build/wnbd/binaries/
+scp_download /workspace/wnbd/vstudio/wnbdevents.xml /$WORKSPACE/build/wnbd/binaries/
+
+mkdir -p $WORKSPACE/build/wnbd/symbols
+scp_download /workspace/wnbd/vstudio/x64/${BUILD_CONFIGURATION}/pdb/driver/* $WORKSPACE/build/wnbd/symbols/
+scp_download /workspace/wnbd/vstudio/x64/${BUILD_CONFIGURATION}/pdb/libwnbd/* $WORKSPACE/build/wnbd/symbols/
+scp_download /workspace/wnbd/vstudio/x64/${BUILD_CONFIGURATION}/pdb/wnbd-client/* $WORKSPACE/build/wnbd/symbols/
+
+#
+# Package the build artifacts into a zip file
+#
+cd $WORKSPACE/build
+zip -r $WORKSPACE/wnbd.zip wnbd
+
+#
+# Upload the the zip file to Chacra
+#
+if [ "$THROWAWAY" = false ]; then
+    # push binaries to chacra
+    chacra_binary="$VENV/chacractl binary"
+    [ "$FORCE" = true ] && chacra_binary="$chacra_binary --force"
+
+    $chacra_binary create ${chacra_binary_endpoint} $WORKSPACE/wnbd.zip
```

Comment:
Let's maybe try `ls $WORKSPACE/wnbd.zip | $chacra_binary create ${chacra_binary_endpoint}` instead.

Reply:
it works now. Thank-you!